cmake_aux/Modules/ARMA_FindOpenBLAS.cmake

```diff
@@ -6,15 +6,20 @@ set(OpenBLAS_NAMES ${OpenBLAS_NAMES} openblas )
 set(OpenBLAS_TMP_LIBRARY)
 set(OpenBLAS_TMP_LIBRARIES)
 
+set(OpenBLAS_SEARCH_PATH ${CMAKE_SYSTEM_LIBRARY_PATH} /lib64 /lib /usr/lib64 /usr/lib /usr/local/lib64 /usr/local/lib /opt/local/lib64 /opt/local/lib)
+
+if(DEFINED ENV{OpenBLAS_DIR})
+  set(OpenBLAS_SEARCH_PATH $ENV{OpenBLAS_DIR} $ENV{OpenBLAS_DIR}/build/lib)
```

Comment:
Using `$ENV{OpenBLAS_DIR}/build/lib` is likely to cause problems, as the user also has to manually tell the run-time linker to look in this folder.  This is an error-prone approach.  I'd strongly prefer not to deal with the extra work and potential problems (ie. validation, bug reports, etc),